CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+### Bug Fixes
+
+* [\#9835](https://github.com/cosmos/cosmos-sdk/pull/9835) Moved capability initialization logic to BeginBlocker to fix nondeterminism issue mentioned in [\#9800](https://github.com/cosmos/cosmos-sdk/issues/9800).
```

Comment:
shall we describe here what a chain has to do to properly apply the fix?

1. get latest SDK
2. add x/capabilities.BeginBlocker to app module manager

Reply:
Also, let's make it in bold in the Change log:
Change in `k.InitAndSeal` **breaks the state machine logic**. It goes under the Stable Release Exception.

Reply:
Oh yikes, is it possible for us on osmosis to gate this breaking behavior to only occur after a certain block height? (Or is it breaking relative to `v0.42.5`)

We would like a binary that can sync from the `v0.42.5` genesis, and only has a behavior change at a given block height.

Reply:
No this is not state machine breaking since the memory store is not committed into app state